WordPress/gutenberg
# Search Block: Refactoring settings panel

## What?
Part of: https://github.com/WordPress/gutenberg/issues/67813

## Why?
As per ticket definition we need to refactor Search Block code to include ToolsPanel instead of PanelBody.


## How?
Refactored Search Block code to include ToolsPanel instead of PanelBody.


## Testing Instructions

- Go to WP admin dashboard
- Edit any post/ page
- Add search block.
- Check settings panel of the search block and verify the change.
- 


## Screenshots or screencast 



|Before|After|
|-|-|
|![beforesearch](https://github.com/user-attachments/assets/15f17b2f-1680-400f-a361-2f118729ae2f)|![aftersearch](https://github.com/user-attachments/assets/2f9beb69-ea7d-4355-8364-39df41348977)|
